2264/3/helpers/comment_rating_counter.rb

```diff
@@ -0,0 +1,50 @@
+require 'net/https'
+require 'uri'
+require 'json'
+
+# Rating Counter
+class CommentRatingCounter
+  URL = 'https://westcentralus.api.cognitive.microsoft.com'.freeze
+  PATH = '/text/analytics/v2.0/sentiment'.freeze
+  KEY = 'de8a560ccae541e08ec1a30dcdd191a4'.freeze
+
+  attr_reader :uri, :documents, :request
```

Comment:
should the class count raiting?
attr_reader should define visible for outside incoming and outgoing params:
I assume it should be texts and ratings
the rest is implementation details - it should be in private section (methods in private sections)